### PR TITLE
Aframe Number Lock Puzzle

### PR DESCRIPTION
**Pull request by amoliksingh**

### CODE REVIEW

File: public/Aframe/samples/floor.js [hunk 1]
```diff
@@ -0,0 +1,20 @@
+AFRAME.registerComponent("floor", {
```
Comment (jaydhulia) [inline public/Aframe/samples/floor.js:1]: Can you remove the components that are unrelated to the numpad from this PR @amoliksingh ?

File: public/Aframe/samples/number_lock.html [hunk 2]
```diff
@@ -0,0 +1,97 @@
...
+        <script src="//cdn.rawgit.com/donmccurdy/aframe-physics-system/v4.0.1/dist/aframe-physics-system.min.js"></script>
+    </head>
+    <body>
+        <a-scene
+            cursor="rayOrigin:mouse"
+            background="color: #ECECEC"
+            physics="debug: false; gravity: -9.81"
+            raycaster="objects: .link"
+        >
+            <a-box
```
Comment (jaydhulia) [inline public/Aframe/samples/number_lock.html:23]: I think all the floor stuff and gravity/physics stuff can be removed from this PR, as we want to compartmentalize this PR to only numpad component, that can be added to our scenes once we start working on those

File: public/Aframe/samples/floor.js (hunk 1 above)
Comment (amoliksingh) [reply public/Aframe/samples/floor.js]: Removed

File: public/Aframe/samples/number_lock.html [hunk 3]
```diff
@@ -0,0 +1,88 @@
...
+                static-body="shape: auto"
+                position="0 -0.52 0"
+                rotation="90 0 0"
+                width="20"
+                height="10"
+                depth="1"
+                floor
+            ></a-box>
+            <a-box
```
Comment (jaydhulia) [inline public/Aframe/samples/number_lock.html:26]: All the boxes are not needed right? Can we get rid of them?

File: public/Aframe/samples/keypad.js [hunk 4]
```diff
@@ -0,0 +1,33 @@
+AFRAME.registerComponent("my_keyboard_controller", {
+    init: function () {
+        this.el.addEventListener("superkeyboardinput", function (event) {
+            if (event.detail.value === "1234") {
```
Comment (jaydhulia) [inline public/Aframe/samples/keypad.js:4]: Instead of hardcoding 1234 in here, can you investigate how to have this in a variable that's passed in when initializing? 
Like you know how in the .HTML file we do `super-keyboard="hand: #mouseCursor;"`, similarly, I am imagining something like `my_keyboard_controller="pass: 1234"`, and then in here using that variable. This generalization will help us get the right checks in with the actual number coming from the backend.

**Review by jaydhulia — DISMISSED** [review]

File: public/Aframe/samples/number_lock.html [hunk 5]
```diff
@@ -0,0 +1,88 @@
...
+                color="white"
+                floor
+            ></a-box>
+            <!-- <a-entity id="keypad-popup-container" visible="true"> -->
+            <a-entity
+                id="keyboard-display"
+                position="0 1.2 -3.9"
+                rotation="0 0 0"
+                scale="3 3 3"
+                super-keyboard="hand: #mouseCursor; imagePath:./; multipleInputs:true; filters: numbers; model: numpad; align: center; maxLength: 4; label: Enter Password; labelColor: black"
```
Comment (jaydhulia) [inline public/Aframe/samples/number_lock.html:76]: What does `hand: #mouseCursor` do?

File: public/Aframe/samples/number_lock.html (hunk 3 above)
Comment (amoliksingh) [reply public/Aframe/samples/number_lock.html]: Removed

File: public/Aframe/samples/number_lock.html (hunk 5 above)
Comment (ahmedhamodi) [reply public/Aframe/samples/number_lock.html]: Yeah I'm also not sure what this does. But it looks like you removed it so I guess it's not needed? Can you confirmed Amolik

**Review by ahmedhamodi — APPROVED** [review]

**Review by jaydhulia — APPROVED** [review]

File: public/Aframe/samples/number_lock.html (hunk 5 above)
Comment (amoliksingh) [reply public/Aframe/samples/number_lock.html]: image
